CHANGELOG.md

```diff
@@ -20,6 +20,27 @@
 * The `MapboxVoiceController` and `RouteVoiceController` now emit `SpeechError`s instead of an `NSError` object. ([#2230](https://github.com/mapbox/mapbox-navigation-ios/pull/2230))
 * Added the `VoiceControllerDelegate.voiceController(_:didFallBackTo:becauseOf:)` method for detecting when the voice controller falls back to `AVSpeechSynthesizer`. ([#2230](https://github.com/mapbox/mapbox-navigation-ios/pull/2230))
 
+### Obsoletion & Removal
```

Comment:
I’d prefer if we moved these entries into the existing topic-based section headers. It’s a bit overwhelming for a developer to see a wall of obsoletion messages, but easier to digest if they know that some are UI-related, others are in Core Navigation, etc.

Reply:
Done, how does that look?